src/courseware/course/sidebar/sidebars/discussions/DiscussionsTrigger.jsx

```diff
@@ -8,11 +8,13 @@ import { useDispatch } from 'react-redux';
 import { useModel } from '../../../../../generic/model-store';
 import { getCourseDiscussionTopics } from '../../../../data/thunks';
 import SidebarTriggerBase from '../../common/TriggerBase';
+import { LAYOUT_RIGHT } from '../../common/constants';
 import SidebarContext from '../../SidebarContext';
 import messages from './messages';
 
 ensureConfig(['DISCUSSIONS_MFE_BASE_URL']);
 export const ID = 'DISCUSSIONS';
+export const LAYOUT = LAYOUT_RIGHT;
```

Comment:
[question]: Do we need this re-export here? This is a general question, I noticed that in some components constants are re-exported. Do we need to fix this?

Reply:
Yes, this is OK because we export it here and use in the sidebar aggregator file.